### DISCUSSION TITLE
[request] logging plugins(http,tcp,udp) add config field

### DISCUSSION THREAD

**Discussion by WALL-E** [Kong/kong#7255] [Ideas and feature requests]
### Summary

logging plugins that should stay a per-protocol: http, tcp, udp, and can come with facilities such as a config field for the serializer and buffer.

**Comment by thibaultcha** [comment]
Thanks for opening this tracking issue! 

Related discussions:
- #1690
- #1731

**Comment by thibaultcha** [comment]
I am classifying this as a proposal and not a "todo" feature since we all need to discuss the pros and cons, and the steps towards implementation (or not).

**Comment by WALL-E** [comment]
Yes, I know
